jobs/config.json

```diff
@@ -187,6 +187,13 @@
   ]
 },
 
+"ci-kubernetes-node-memcg-serial": {
+  "scenario": "kubernetes_kubelet",
+  "args": [
+    "--properties=test/e2e_node/jenkins/jenkins-memcg-serial.properties"
+  ]
+},
+
```

Comment:
do you want `ci-kubernetes-node-memcg-serial` or `kubernetes-e2e-gci-memcg-serial` or both? They are pretty different :-) Let me understand it correctly first.